.circleci/config.yml

```diff
@@ -5,19 +5,6 @@ orbs:
 workflows:
   package-and-push-chart-on-tag:
     jobs:
-      - architect/push-to-app-catalog:
```

Comment:
Does it mean we don't want to let customers install it on their WCs?

Reply:
Same as Loki and Mimir yes